lib/helpers.js

```diff
@@ -17,6 +17,46 @@ import Tx from './transaction';
 
 import { BLOCKS_PER_PERIOD } from './constants';
 
+function getLowerCase(value) {
+  if (value) { return value.toLowerCase(); }
+  return value;
+}
+
+const fromRaw = (u) => ({
+  output: u.output,
+  outpoint: Outpoint.fromRaw(u.outpoint),
+})
+
+export class leapEthers {
+/** 
+ * @param { import("ethers").providers.JsonRpcProvider } provider
+*/
+  constructor(provider) {
+    this.provider = provider
+  }
+
+  getUnspent = async (...params) => { 
+    const unspent = await this.provider.send("plasma_unspent", params.map(p => getLowerCase(p)))
+    return unspent.map((u) => fromRaw(u))
+  }
+  
+  getUnspentAll = async () => { 
```

Comment:
Why you defined methods like class props, not as plain methods? If because of `this`, i don't think library should be concern about such matters

Reply:
I just like the syntax, It's not necessary.  Thank you for the quick review; i will address the points you have raised.